plugin/public/block.php

```diff
@@ -148,14 +168,14 @@ function display( $attr ) {
 		$id   = $attr['fileId'];
 
 		if ( isset( $attr['embedWidth'] ) ) {
-			$size .= 'width:' . $attr['embedWidth'] . 'px; ';
+			$size .= 'width:' . parse_dimension( $attr['embedWidth'] ) . '; ';
 		}
 
 		if ( isset( $attr['embedHeight'] ) ) {
-			$size .= 'height:' . $attr['embedHeight'] . 'px; ';
+			$size .= 'height:' . parse_dimension( $attr['embedHeight'] ) . '; ';
 		}
 
-		$result = '<iframe src="https://drive.google.com/file/d/' . $id . '/preview" style="' . $size . 'border:0;"></iframe>';
+		$result = '<iframe class="sgdd-embedded-file" src="https://drive.google.com/file/d/' . $id . '/preview" style="' . $size . 'border:0;"></iframe>';
```

Comment:
Is this change related to the PR? I'm a bit confused...

Reply:
Ah, that's the fourth point, I see...

Reply:
Umm, then the question becomes "Why?" The class isn't used anywhere else...

Reply:
Yes - it is not using by plugin :) It is possibility for plugin user for customization - i.e. add some margin. Without class name there is no chance to customize look of frame via external CSS.

Reply:
Ok, makes sense now.